src/api/controller/api/field.js

```diff
@@ -131,6 +124,16 @@ export const importFields = getUploadedFieldsImpl => async ctx => {
     ctx.status = 200;
 };
 
+export const reorderField = async ctx => {
+    const { fields } = ctx.request.query;
+    ctx.body = await Promise.all(
+        Object.values(fields).map((name, index) =>
+            ctx.field.findOneAndUpdate({ name }, { $set: { position: index } }),
```

Comment:
I would ensure some validations here IMO
Like, at least, check that the URI field is always at position 0
(if its needed)